src/dash-table/components/Table/props.ts

```diff
@@ -301,6 +301,7 @@ export interface IProps {
     tooltip_conditional: ConditionalTooltip[];
 
     active_cell?: ICellCoordinates;
+    cell_selectable?: boolean;
```

Comment:
New optional prop. If false, user can't select cells and `active_cell` and `selected_cells` will always be treated as null/empty.